crates/hive-utils/scripts/start.sh

```diff
@@ -8,6 +8,7 @@ export PROXY_ACCOUNT_CLASS_HASH=$(starkli class-hash ${MADARA_PATH}/cairo-contra
     --rpc-external \
     --rpc-methods=unsafe \
     --rpc-cors=all \
+    --disable-madara-configs \
```

Comment:
Since we're copying all the genesis assets into the local Madara path, we need to add this flag to use that